boris-http/src/Boris/Http/Route.hs

```diff
@@ -407,16 +400,30 @@ route store authentication buildx logx projectx mode = do
       log'' <- liftError Store.renderFetchError $
         Build.logOf store logx (BuildId buildId)
 
-      case log'' of
-        Nothing -> do
+      withAccept $ \case
+        AcceptHTML -> do
           Spock.setStatus HTTP.notFound404
           Spock.html "TODO: 404 page."
-        Just logs ->
-          Spock.stream $ \send flush ->
-            runConduit $
-              logs =$=
-                CL.mapM_ (\t -> send (fromByteString t) >> flush)
-
+--          case log'' of
+--            Nothing -> do
+--              Spock.setStatus HTTP.notFound404
+--              Spock.html "TODO: 404 page."
+--            Just b ->
+--              View.render $ View.build b
+        AcceptJSON -> do
+          case log'' of
+            Nothing -> do
+              Spock.setStatus HTTP.notFound404
+              Spock.json ()
+            Just logs -> do
+              Spock.json $ ApiV1.GetLogs t
+              where
+                t :: IO ()
+                t = runConduit $ logs =$= CL.mapM_ (TextIO.putStrLn . Text.decodeUtf8)
```

Comment:
For now, this is my best way of getting to the data. I think I have to stay in `IO` because of the type of `logs`. Should this change? See also https://github.com/markhibberd/boris/pull/4/files#r169225278